website/www/site/content/en/contribute/release-guide.md

```diff
@@ -283,6 +283,8 @@ After following one of these processes you should manually update `CHANGES.md` o
      [version.py](https://github.com/apache/beam/blob/release-2.6.0/sdks/python/apache_beam/version.py#L21),
      [build.gradle](https://github.com/apache/beam/blob/release-2.6.0/runners/google-cloud-dataflow-java/build.gradle#L39),
      [gradle.properties](https://github.com/apache/beam/blob/release-2.16.0/gradle.properties#L27)
+     
+**ATTENTION:** The Release Candidate Number ``rc_num`` is an optional parameter to add the version of the release candidate.
```

Comment:
I found this thread discussing how to publish unreleased artifacts and whether we can advertise them: https://lists.apache.org/thread.html/f071f8ab9f115636b9e6a6cabcfccbe2bb980d4394fe5581c59a4db6%40%3Cdev.beam.apache.org%3E. We should make sure that the process aligns with the Apache requirements, and it would a good idea to write up a short summary for the new process, send it to dev@ and make sure there are no concerns.